src/classes/issues-processor.ts

```diff
@@ -268,6 +317,130 @@ export class IssuesProcessor {
       );
     }
 
+    if (this.options.mlflow) {
+      const {startDate, daysBeforeReplyReminder, daysBeforeAssigneeReminder} =
+        this.options;
+      if (
+        issue.milestone ||
+        issue.isStale ||
+        (startDate &&
+          !isDateMoreRecentThan(
+            new Date(issue.created_at),
+            new Date(startDate)
+          ))
+      ) {
+        return;
+      }
+
+      const reminderToMaintainers = 'Reminder to MLflow maintainers';
+      if (issue.isPullRequest) {
+        // TODO
+      } else {
+        const hasClosingPr = issue.labels.some(
+          ({name}) => name === 'has-closing-pr'
+        );
+        if (hasClosingPr) {
+          issueLogger.info('This issue has a closing PR');
+          return;
+        }
+        const comments = await this.listIssueComments(issue, issue.created_at);
+        issueLogger.info(
+          `The number of comments in this issue: ${comments.length}`
+        );
+
+        issueLogger.info(
+          `Assignees on this issue: ${issue.assignees.map(({login}) => login)}`
+        );
+
+        if (comments.length > 0) {
```

Comment:
What happens if a user opens an issue and then adds a second comment to it. Looks like we'll never triage it. Can we remove this comment length check?

Reply:
Makes sense, will remove it